File: exercicios/1789/1789.py
while True:
    try:
        n = int(input())
        x = input().split()
        m = 0

        for i in range(n):
            if int(x[i]) > m:
                m = int(x[i])
        
        if m < 10:
            n = 1
        elif m >= 10 and m < 20:
            n = 2
        else:
            n = 3
        print(n)
    
    except EOFError:
        break

File: exercicios/JavaScript/1789/1789.js
var input = require('fs').readFileSync('stdin','utf8')
var lines = input.split('\n')

while (lines.length > 0) {
    m = 0;
    classific = 0;

    var n = lines.shift()

    var v = lines.shift().split(' ')

    
    for(let i=0; i<n; i++){
        var val = parseInt(v[i])
        
        if(val>m){
            m=val
        }

        if(m < 10){
            classific = 1;
        }else if(m >= 10 && m < 20){
            classific = 2;
        }else{
            classific = 3;
        }
    }
    console.log(classific);
}